### PR TITLE
Khmer lang support

### PR DESCRIPTION
**Pull request by davychhouk**
Add Khmer language support.

Translation Reference: [Official Country Name Translation by the National Commission of Khmer Language](http://iric.gov.kh/names-of-countries-and-capitals-in-the-world-by-national-committee-of-khmer-language)

### CODE REVIEW

**Comment by davychhouk** [comment]
Hi, @michaelwittig! 
Please have a look again. 
Thanks :)

File: langs/kh.json [hunk 1]
```diff
@@ -0,0 +1,255 @@
+{
```
Comment (michaelwittig) [inline langs/kh.json:1]: could you please remove this file. this lib only contains ISO 639-1 codes.

File: entry-node.js [hunk 2]
```diff
@@ -46,7 +46,9 @@ var locales = [
   require("./langs/it.json"),
   require("./langs/ja.json"),
   require("./langs/ka.json"),
+  require("./langs/kh.json"),
```
Comment (michaelwittig) [inline entry-node.js:49]: could you please remove this line. this lib only contains ISO 639-1 codes.

File: README.md [hunk 3]
```diff
@@ -79,7 +79,9 @@ console.log(countries.getNames("en", {select: "official"})); // { 'AF': 'Afghani
 - `id`: Indonesian
 - `ja`: Japanese
 - `ka`: Georgian
+- `kh`: Khmer
```
Comment (michaelwittig) [inline README.md:82]: could you please remove this line. this lib only contains ISO 639-1 codes.

**Review by michaelwittig — CHANGES_REQUESTED** [review]

**Comment by davychhouk** [comment]
Hi @michaelwittig.

This is weird. I'm sorry. There must be sth wrong with my git thing.
In my local is clean, but after pull from the master branch, it becomes a mess.

I removed all of these again. Thanks.

**Comment by michaelwittig** [comment]
no worries! thanks for your work!

**Pull request merged**